src/completions.jl

```diff
@@ -12,40 +31,27 @@ handle("completions") do data
     startRow || 0,
     column || 1,
     # configurations
+    is_fuzzy || true,
```

Comment:
Do you think fuzzy completions should be default ?

Reply:
Not here. Old julia-client versions probably can't display those correctly, right?

Reply:
This PR is now compatible with the last version at least (only the fuzziness changes).
Maybe julia-client≦0.11 won't be compatible with this PR but it's same for the current master.